src/app/core/views/DeactivationView/DeactivationView.tsx

```diff
@@ -41,9 +42,10 @@ class DeactivationView extends React.Component<DeactivationViewProps> {
   };
 
   ConfirmDeactivateUser = (token: string) => {
-    const authClient = SdkFactory.getInstance().createAuthClient();
+    const authClient = SdkFactory.getNewApiInstance().createAuthClient();
+    const xNewToken = localStorageService.get('xNewToken') ?? undefined;
```

Comment:
maybe could change the name for `userToken` to differentiate from the deactivation token